conda/recipes/libcucim/meta.yaml

```diff
@@ -55,15 +55,14 @@ requirements:
   host:
     - cuda-version ={{ cuda_version }}
     - cuda-cudart-dev
-    - libcufile-dev     # [linux64]
+    - libcufile-dev
     - libnvjpeg-dev
-    - libnvjpeg-static
```

Comment:
nvJPEG is now dynamically linked to in the Conda case. Hence no need for the static library